docs/synchronizations-faq.md

```diff
@@ -0,0 +1,33 @@
+# Synchronizations FAQ
+##
+
+In Private Packagist synchronizations is how you keep packages, teams, their members, and access permissions in sync with your version control system like GitHub, GitLab or Bitbucket.
+
+You can find below answers for frequently-asked questions concerning synchronizations.
+
+## How to migrate my synchronization from GitHub to Bitbucket?
+
+1. Add the new synchronization from Bitbucket.
+2. Delete the packages that were created from the GitHub synchronization. You can filter by the GitHub synchronization on the package list page and then you can delete the packages from that list.
+3. Make sure the new synchronization from Bitbucket successfully run. In case the old synchronization was marked as primary, you need to promote the new synchronization as primary otherwise you won't be able to delete it. After switching the primary synchronization, a new owner team will be fetched from Bitbucket. To make sure you keep your access, you'll have to go to your profile page and connect your Private Packagist account with Bitbucket.
```

Comment:
We don't fetch a new owner team but instead after the Private Packagist Owners team will only consist of admins of the new Bitbucket workspace and no longer the GitHub organization. This already gets rather long. I wonder whether we should have a new question entry about primary syncs explaining what makes a primary sync a primary sync and what happens if you promote a different sync to primary. Then we can link to the primary article and don't have to repeat everything in all questions?

Reply:
ok, I added a section explaining the primary sync